src/components/Text/Text.tsx

```diff
@@ -110,7 +110,8 @@ export const Text: React.FC<Props> = ({
     styles.frame,
     css`
       ${options.styles ? replaceVariables(options.styles) : ''}
```

Comment:
`options.styles` applied to both the row and the panel. Is it correct?

Reply:
Right, my bad.
We don`t need apply options.styles inside Text component now
Updated. 
please review